app/Main.hs

```diff
@@ -44,7 +44,7 @@ usageDoc = Just $ mconcat
 contracts :: ContractRegistry
 contracts = ContractRegistry $ Map.fromList
   [ "BaseDAO" ?:: ContractInfo
-    { ciContract = DAO.baseDaoContract
+    { ciContract = DAO.baseDAOContract
```

Comment:
We have a convention to avoid 4 capital letters in a row because it's usually hard to read. Even though `baseDAOContract` is more correct, `baseDaoContract` looks a bit more readable (it's easier to see the boundary between "DAO" and "Contract"). I wouldn't object if it was called `baseDAOContract` from the beginning, but since it's already called `baseDaoContract`, I think it's better to keep this name.

Reply:
I see, that makes sense, Ivan. I've updated this back.